SortApp/SortApp.Presentation/App.xaml.cs

```diff
@@ -1,17 +1,26 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Data;
-using System.Linq;
-using System.Threading.Tasks;
 using System.Windows;
+using SortApp.ViewModel;
 
 namespace SortApp.Presentation
 {
-	/// <summary>
-	/// Interaction logic for App.xaml
-	/// </summary>
-	public partial class App : Application
-	{
-	}
+    /// <summary>
+    /// Interaction logic for App.xaml
+    /// </summary>
+    public partial class App : Application
+    {
+        /// <summary>
+        /// Sets the data context for MainWindow.
+        /// </summary>
+        /// <owner>Oleh Petrenko</owner>
+        public App()
+        {
+            var mw = new SortApp.Presentation.MainWindow

```

Comment:
SortApp.Presentation part is redundant, just add corresponding using (or no using at all as it's the same namespace